specification.md

```
@@ -0,0 +1,231 @@
+# Specification
+
+## Interface Definition
+
+### Operations
+
+#### jsonrpc2:handle/2
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)